desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -12,10 +12,12 @@ module.exports = class BidForm extends ErrorHandlingForm
 
   timesPolledForBidPlacement: 0
   maxTimesPolledForBidPlacement: sd.MAX_POLLS_FOR_MAX_BIDS
-  errors:
+  errors: -> {
     "Sale Closed to Bids": "Sorry, your bid wasn't received before the auction closed."
+    "Live Bidding has Started": => "Live bidding on this sale has begun. <br>To continue bidding, please <a href=\"#{@model.liveAuctionUrl()}\">join our Live Auction Room</a>."
```

Comment:
#minor but this wording is a little odd. Maybe something like "Your bid was not received before the auction went live. To continue bidding, please (link) join the live auction (end link)."